src/js/controllers/index.js

```diff
@@ -152,6 +153,16 @@ angular.module('copayApp.controllers').controller('indexController', function($r
         }
       }
 
+      if (self.isNode) {
+        profileService.checkLatestRelease(function(err, version) {
+          if (err) {
+            $log.warn(err);
+            return;
+          }
+          self.newVersion = 'There is a new version of Copay (' + version + '). Please upgrade';
```

Comment:
you should use `gettext` for this string instead of a `translate` filter in the view.